samples/charts/area/line-datasets.html

```diff
@@ -122,7 +122,7 @@
 				filler: {
 					propagate: false
 				},
-				samples_filler_analyser: {
+				'samples-filler-analyser': {
```

Comment:
You also need to change the [plugin id](/chartjs/Chart.js/blob/master/samples/charts/area/analyser.js#L7)